src/config.php

```diff
@@ -9,6 +9,17 @@
         // Specifies whether to enable this extension.
         'enable' => env('LQEXT_ENABLE_TXN_AWARE', false),
 
+        // Specifies if the environment is testing
+        'testing' => env('LQEXT_ENV_TESTING', false),
```

Comment:
This is not needed 
This being laravel package you can just use APP_ENV=testing or contract method- https://github.com/illuminate/contracts/blob/master/Foundation/Application.php#L83